### PR TITLE
Add PASSKEY and CPUS

### PR DESCRIPTION
**Pull request by Shihta**

### CODE REVIEW

**Review by robertnetz — APPROVED** [review]
thanks for your contribution! 👍

File: Dockerfile [hunk 1]
```diff
@@ -5,20 +5,15 @@ ENV TEAM=0
 ENV POWER=medium
 ENV DOWNLOAD=https://download.foldingathome.org/releases/public/release/fahclient/debian-stable-64bit/v7.5/fahclient_7.5.1_amd64.deb
 
-RUN apt-get update
-RUN apt-get -y install bzip2 wget
+RUN apt-get update && apt-get -y install bzip2 wget openvpn
```
Comment (robertnetz) [inline Dockerfile]: why do you need openvpn here?

Comment (Shihta) [reply Dockerfile]: My Internet status is a bit strange in some of my environment.
I can't download WU if I don't connect to another jump server via openvpn.
Should I remove it?

Comment (robertnetz) [reply Dockerfile]: Yes, please :-)

**Review by robertnetz — APPROVED** [review]

**Pull request merged**